boilerplate/openshift/golang-osd-operator/prow-config

```diff
@@ -0,0 +1,176 @@
+#!/bin/bash -e
+
+REPO_ROOT=$(git rev-parse --show-toplevel)
+source $REPO_ROOT/boilerplate/_lib/common.sh
+
+cmd=${0##*/}
+
+release_repo=openshift/release
+
+usage() {
+    cat <<EOF
+Usage: $cmd [PATH_TO_RELEASE_CLONE]
+
+Creates a delta in $release_repo standardizing prow configuration for a
+boilerplate consumer. Must be invoked from within a local clone of a repository
+already subscribed to the $CONVENTION_NAME convention.
+
+Parameters:
+    PATH_TO_RELEASE_CLONE   File system path to a local clone of
+                            https://github.com/$release_repo. If not
```

Comment:
Should we specify the exact repo here? I assume people running this target will know what it is but just a thought.

Reply:
We're in an interpolating `cat`, so this will expand the `$release_repo` variable as defined on L8. (IOW it will specify the exact repo.)

Reply:
```
[efried@efried boilerplate]$ ./boilerplate/openshift/golang-osd-operator/prow-config -h
Usage: prow-config [PATH_TO_RELEASE_CLONE]

Creates a delta in openshift/release standardizing prow configuration for a
boilerplate consumer. Must be invoked from within a local clone of a repository
already subscribed to the openshift/golang-osd-operator convention.

Parameters:
    PATH_TO_RELEASE_CLONE   File system path to a local clone of
                            https://github.com/openshift/release. If not
                            specified, the repository will be cloned in a
                            temporary directory.
```